aidants_connect_carto_api/models.py

```diff
@@ -0,0 +1,150 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+from django.core.validators import RegexValidator
+
+
+class Place(models.Model):
+    LANGUAGE_CHOICES = [
+        ("fr", "Français"),
+        ("en", "Anglais"),
+        ("sl", "Language des signes"),
+    ]
+    EQUIPEMENT_CHOICES = [
+        ("wifi", "WiFi"),
+        ("ordinateur", "Ordinateur"),
+        ("scanner", "Scanner"),
+        ("imprimante", "Imprimante"),
+        # "Autre"
+    ]
+    HANDICAP_CHOICES = [
+        ("handicap moteur", "Handicap moteur"),
+        ("handicap visuel", "Handicap visuel"),
+        ("handicap auditif", "Handicap auditif"),
+        ("handicap mental", "Handicap intellectuel ou psychique"),
+        # "Maladie invalidante",
+        # "Mobilité limitée"
+    ]
+    PAYMENT_CHOICES = [
+        ("especes", "Espèces"),
+        ("carte bancaire", "Carte bancaire"),
+        ("cheque", "Chèque"),
+        ("aptic", "Chèque APTIC"),
+        ("cif", "Congé individuel de formation (CIF)"),
+        # "CRP",
+        # "AFPE"
+    ]
+
+    ## basics
+    name = models.CharField(max_length=300, help_text="Le nom du lieu")
+
+    ## location
+    address_raw = models.CharField(max_length=300, help_text="L'adresse brute, complète")
+    address_housenumber = models.CharField(max_length=5, help_text="Le numéro avec indice de répétition éventuel (bis, ter, A, B)")
+    address_street = models.CharField(max_length=150, help_text="Le nom de la rue")
+    address_postcode = models.CharField(max_length=5, help_text="Le code postal")
+    address_citycode = models.CharField(max_length=5, help_text="Le code INSEE de la commune")
+    address_city = models.CharField(max_length=150, help_text="Le nom de la commune")
+    # address_context = models.CharField(max_length=150, help_text="n° de département, nom de département et de région")
+    latitude = models.FloatField(help_text="La latitude (coordonnée géographique)")
+    longitude = models.FloatField(help_text="La latitude (coordonnée géographique)")
+    itinerant = models.BooleanField(default=False, help_text="Le lieu est-il itinérant ?")
+    
```

Comment:
Est-ce qu'on considère à ce stade comme prématuré / inutile d'utiliser [`GeoDjango`](https://docs.djangoproject.com/en/3.0/ref/contrib/gis/) pour avoir des [modèles géographiques](https://docs.djangoproject.com/en/3.0/ref/contrib/gis/tutorial/#geographic-models) ? :)

Reply:
bonne question. j'ai hésité honnêtement, mais j'ai pas assez d'experience et le besoin ne s'est pas fait sentir yet. mais GeoDjango + PostGIS pas mal de choses sont out-of-the-box.

Reply:
Oui, d'un côté à partir du moment où on stocke latitude et longitude, on sent bien que ça pourrait être utilisé pour faire des trucs genre _trouve-moi le lieu le plus proche de moi avec un scanner_, d'un autre côté, comme dirait notre Glorieuse Leader Suprême : YAGNI ?